spec/services/geocoder_service_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe GeocoderService do
+  context 'happy path' do
```

Comment:
this is good. 

you might consider a few changes to be a little more consistent with Rails Convention when writing tests. 

```
describe "#get_coordinates"
  context "success" do
    let(:location) { "83-1 64th St Kenosha, WI" }
    let(:response) { GeocoderService.get_coordinates(location) }

    it "returns the lat and long as a Hash" do
      expect(response).to be_a(Hash)
    end
 
    it "returns properly formatted and the correct lat and long" do
      data = response[:results][0]
      expect(data).to be_a(Hash)
      lat_long = data[:locations][0][:latLng]
      expect(lat_long).to have_key(:lat)
      expect(lat_long).to have_key(:lng)
      expect(lat_long[:lat]).to be_a(Float)
      expect(lat_long[:lng]).to be_a(Float)
    end
  end

  context "failure" do
   # add sad path tests here
  end
end
```

Reply:
Okay! Noted.